lib/cinch/plugins/wordgames/game.rb

```diff
@@ -35,14 +32,25 @@ def number_of_guesses_phrase
     end
 
   protected
-    def guess_correct?(word, response)
-      if @word == word
-        response.game_won
-        true
+    def blank?(bound)
+      bound == Blank_str
+    end
+
+    def evaluate_guess!(word)
```

Comment:
As that Game#Guess! has the potential to modify the Game instance itself, I decided it was noisy, notice `@lower_bound` and `@upper_bound` are liable to mutation.  More importantly, I didn't want confusion between a `guess` variable and a `guess` method invocation so banging seemed appropriate (I tend to not go out of my way to mark noisiness in my ruby code though, this just happened to work into the refactor).  

I'm glad you spotted `guess!` and `evaluate_guess` I saw that too and was tempted to change it, but as per https://en.wikipedia.org/wiki/Single_responsibility_principle I decided the evaluation of a guess (the CPU crunching of it, that is) should not rely on a buffer counting the number of times it's been run.  

Perhaps this will lead to greater flexibility down the road as we figure out how to define an abstract class that may not involve counting guesses at all for some future game.